src/components/menu/SelectMenuLinkItem.md

```diff
@@ -1,14 +1,25 @@
+### Description
+
+**Menu (ARIA-compliant)**
+This component is used to render a ARIA-compliant menu.
+The component handles adding aria attributes, keyboard navigation, and focus.
+
+The `<MenuLinkItem>` component is required to wrap anchor tags since there are special ARIA rules for those.
+The `<SelectMenuLinkItem>` component is for items that can be selected with a checkmark.
+
+Following the following standards: [WAI-ARIA Menu](https://www.w3.org/TR/wai-aria-practices-1.1/#menu)
+
 ### Examples
 
 ```
 const Menu = require('box-ui-elements/es/components/menu').Menu;
-const SelectMenuLinkItem = require('box-ui-elements/es/components/menu').SelectMenuLinkItem;
```

Comment:
why are some of these removed but others are left? can we make this consistent?

Reply:
functional components need to be imports somewhere while class components dont when they are only used once

Reply:
Some of the components in the Readme file is throwing error 'Identifier xxx has already been declared'. However, after removing those, it will throw undefined error. As a result, we need to move the import to the styleguidist.setup.js file. For component that is not throwing error, I tried to do the same thing just for consistence, but it will make the component render differently so that I only updated component import that is causing issue.